metrics/firebase/test/firestore/rules/test_utils/helpers.js

```diff
@@ -0,0 +1,44 @@
+const {
```

Comment:
Does this look more like Util - test_app_utils?
https://stackoverflow.com/questions/12192050/what-are-the-differences-between-helper-and-utility-classes